src/Employment/WorkContract_v1.0.py

```diff
@@ -0,0 +1,340 @@
+    )
+
+
+class EmployeeInfo(CamelCaseModel):
+    name: str = Field(
+        ...,
+        title="Name",
+        max_length=250,
+        description="The name of the employee",
+        examples=["Tom Williams"],
+    )
+    street_address: str = Field(
+        ...,
+        title="Street Address",
+        max_length=40,
+        description="The street name of the employee contact address",
+        examples=["49 Featherstone Street"],
+    )
+    postal_code: str = Field(
+        ...,
+        title="Postal Code",
+        max_length=10,
+        description="The postal code of the employee address",
+        examples=["EC1Y 8SY"],
+    )
+    city: str = Field(
+        ...,
+        title="City",
+        max_length=40,
+        description="The city of the employee address",
+        examples=["London"],
+    )
+    country: Optional[str] = Field(
+        ...,
+        title="Country",
+        max_length=3,
+        description="The country code of the employee address in Alpha-3 format",
+        examples=["GBR"],
+    )
+    signature_date: date = Field(
+        ...,
+        title="Signature Date",
+        description="The date of the employee signature for the work contract",
+        examples=[datetime.datetime(2023, 8, 27)],
+    )
+
+
+class TermsOfWork(CamelCaseModel):
+    employment_start: date = Field(
+        ...,
+        title="Employment Start",
+        description="The start date of the employment",
+        examples=[datetime.datetime(2023, 11, 7)],
+    )
+    employment_end: date = Field(
+        ...,
+        title="Employment End",
+        description="The end date of the employment",
+        examples=[datetime.datetime(2024, 2, 19)],
+    )
+    grounds_for_fixed_term: Optional[str] = Field(
+        None,
+        title="Grounds for Fixed Term",
+        max_length=250,
+        description="The reasoning if the contract is fixed term",
+        examples=["The winter time tourism in Lapland"],
+    )
+    work_duties: str = Field(
+        ...,
+        title="Work Duties",
+        max_length=400,
+        description="The work duties of the employee",
+        examples=["Hotel guest reception"],
+    )
+    work_conditions: Optional[str] = Field(
+        None,
+        title="Work Conditions",
+        max_length=400,
+        description="The special conditions regarding the work duties, e.g. work for another party than the employer",
+        examples=["The work duties done for Lapland Hotels"],
+    )
+    industry: Optional[str] = Field(
+        None,
+        title="Industry",
+        max_length=10,
+        description="The work industry based on the Statistical classification of economic activities in the European "
+        "Community, abbreviated as NACE.",
+        examples=["79.1"],
+    )
+    locations: List[str] = Field(
+        None,
+        title="Locations",
+        description="The region in Finland where the work is done",
+        examples=["Rovaniemi", "Lapland"],
```

Comment:
This is supposed to be one example.
```suggestion
        examples=["Rovaniemi, Lapland"],
```